addons/addon-base-raas/packages/base-raas-services/lib/data-egress/data-egress-service.js

```diff
@@ -190,6 +194,13 @@ class DataEgressService extends Service {
     }
 
     const egressStoreInfo = await this.getEgressStoreInfo(environmentId);
+    if (!egressStoreInfo) {
+      await this.audit(requestContext, {
+        action: 'terminated-egress-store',
+        body: 'No egress store found to be terminated',
+      });
+      return null;
```

Comment:
Do we need a unit test here?

Reply:
tests added in commit: 2a80047064bb961f9e8b9fa14643cb65033161f7